src/components/Input/index.examples.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import {keys} from 'lodash'
+import {storiesOf} from '@kadira/storybook'
+import Input from './index'
+import PasswordTooltip from '../Tooltip/PasswordTooltip'
+
+const decoratorClasses = 'flex flex-column content-center justify-around flex-wrap items-center vh-100 bg-navy'
+
+// Empty Field
+storiesOf('Input')
+  .addDecorator((story) => (<div className={decoratorClasses}>{story()}</div>))
+  .addWithInfo('Empty Field', () => (
+    <Input />
+  ))
+
+// Field with Placeholder
+storiesOf('Input')
+  .addDecorator((story) => (<div className={decoratorClasses}>{story()}</div>))
+  .addWithInfo('Field with Placeholder', () => (
+    <Input placeholder='Placeholder' />
+  ))
+
+// Disabled Field
+storiesOf('Input')
+  .addDecorator((story) => (<div className={decoratorClasses}>{story()}</div>))
+  .addWithInfo('Disabled Field', () => (
+    <Input placeholder='Disabled' disabled />
+  ))
+
+// Field with Error Message
+storiesOf('Input')
+  .addDecorator((story) => (<div className={decoratorClasses}>{story()}</div>))
+  .addWithInfo('Field with Error', () => (
+    <Input
+      value='Error'
+      required
+      error
+      errorMsg='Password must contain at least 8 characters.'
+      icon='error'
+    />
+  ))
+
+// Success Field
+storiesOf('Input')
+  .addDecorator((story) => (<div className={decoratorClasses}>{story()}</div>))
+  .addWithInfo('Password Success Field', () => (
+    <Input
+      value='Password'
+      required
+      type='password'
+      icon='success'
+    />
+  ))
+
+// Input Block with Password Tooltip
+storiesOf('Input')
```

Comment:
I'm pretty sure you can combine all the `storiesOf('Thing')` anytime `Thing` is the same - this also removes the need to add the same decorator over and over; so for example, lines 10-44 could be reduced to:

```
storiesOf('Input')
  .addDecorator(...)
  .addWithInfo('Empty field', ...)
  .addWithInfo('Field with placeholder', ...)
  .addWithInfo('Disabled field', ...)
  .etc
```